pytorch_lightning/trainer/data_loading.py

```diff
@@ -67,224 +67,6 @@ class TrainerDataLoadingMixin(ABC):
     dev_debugger: InternalDebugger
     call_hook: Callable
 
-    def _worker_check(self, dataloader: DataLoader, name: str) -> None:
```

Comment:
Can you put the code where it was? This diff is very hard to review.

If moving the position of the code was intended, then it should be in 2 PRs. 1 modifying the code and another one moving it